submissions/OlexiyDobroskok/task_js_dom/script.js

```diff
@@ -0,0 +1,145 @@
+    photo: "./img/Wolves of the Calla.jpg",
+    alt: "Illustration for Stephen King's The Dark Tower V: Wolves of the Calla",
+  },
+  {
+    id: "dt6",
+    title: "The Dark Tower VI: Song of Susannah",
+    quote:
+      "“There are forces at work ... not all ... are working to keep us away from the Tower. ”",
+    about:
+      "The Wolves have been defeated, but our tet faces yet another catastrophe. Susannah Dean’s body has been usurped by a demon named Mia who wants to use Susannah’s mortal form to bear a demon child. Stealing Black Thirteen, Mia has traveled through the Unfound Door to 1999 New York where she plans to give birth to her chap, a child born of two mothers and two fathers who will grow up to be Roland’s nemesis. With the help of the time-traveling Manni, Roland and Eddie plan to follow Susannah while Father Callahan and Jake will find Calvin Tower, owner of the vacant lot where a magical rose grows: a rose that must be saved at all costs.",
+    photo: "./img/Song of Susannah.jpg",
+    alt: "Illustration for Stephen King's The Dark Tower VI: Song of Susannah",
+  },
+  {
+    id: "dt7",
+    title: "The Dark Tower VII: The Dark Tower",
+    quote:
+      "“Fault always lies in the same place, my fine babies: with him weak enough to lay blame.”",
+    about:
+      "The final volume sees gunslinger Roland on a roller-coaster mix of exhilarating triumph and aching loss in his unrelenting quest to reach the dark tower. Roland's band of pilgrims remains united, though scattered. Susannah-Mia has been carried off to New York to give birth, Terrified of what may happen.",
+    photo: "./img/The Dark Tower.jpg",
+    alt: "Illustration for Stephen King's The Dark Tower VII: The Dark Tower",
+  },
+  {
+    id: "dt8",
+    title: "The Dark Tower: The Wind Through the Keyhole",
+    quote: "“Balls to your sorry ... Ka works and the world moves on. ”",
+    about:
+      "We join Roland and his ka-tet as a ferocious storm halts their progress along the Path of the Beam. As they shelter from the screaming wind and snapping trees, Roland tells them not just one strange tale, but two – and in doing so sheds fascinating light on his own troubled past.",
+    photo: "./img/The Wind through the Keyhole.jpg",
+    alt: "Illustration for Stephen King's The Dark Tower: The Wind Through the Keyhole",
+  },
+];
+
+const contentContainer = document.querySelector(".content");
+const sidebar = document.createElement("aside");
+sidebar.className = "sidebar";
+contentContainer.prepend(sidebar);
+sidebar.prepend(...addButtons());
+
+function addButtons() {
+  const buttonList = [];
+  darkTowerSeries.forEach((el, index) => {
+    const button = document.createElement("button");
+    button.className = "sidebar__button";
+    button.id = "dt" + index;
+    button.innerHTML = darkTowerSeries[index].title;
```

Comment:
What is the mean name 'el'? It is better named as 'serie'
```suggestion
    button.innerHTML = el.title;
```